be/src/exprs/CMakeLists.txt

```diff
@@ -26,7 +26,14 @@ set(LIBRARY_OUTPUT_PATH "${BUILD_DIR}/src/exprs")
 set(EXECUTABLE_OUTPUT_PATH "${BUILD_DIR}/src/exprs")
 
 set(EXPR_FILES 
-  agg/aggregate_factory.cpp
+  agg/factory/aggregate_factory.cpp
+  agg/factory/aggregate_resolver_1.cpp
+  agg/factory/aggregate_resolver_2.cpp
+  agg/factory/aggregate_resolver_3.cpp
+  agg/factory/aggregate_resolver_4.cpp
+  agg/factory/aggregate_resolver_5.cpp
+  agg/factory/aggregate_resolver_6.cpp
+  agg/factory/aggregate_resolver_7.cpp
```

Comment:
could we give these files a more meaningful name…

Reply:
Good advice. I could register each function independently.